src/devices/nodon.ts

```diff
@@ -147,6 +147,14 @@ const definitions: Definition[] = [
             await reporting.currentSummDelivered(ep);
         },
     },
+    {
+        zigbeeModel: ['STPH-4-1-00'],
+        model: 'STPH-4-1-20',
+        vendor: 'NodOn',
+        description: 'Temperature & humidity sensor',
+        extend: [batteryPercentage(), temperature({'endpoints': ['1']}), humidity({'endpoints': ['1']})],
```

Comment:
```suggestion
        extend: [batteryPercentage(), temperature(), humidity()],
```

Does this also work?

Reply:
Yes